macintacos/vspacecode-config.json

```diff
@@ -552,128 +572,449 @@
-              "command": "csv.preview",
-              "key": "v",
-              "name": "CSV Preview",
+              "command": "editor.action.formatDocument",
+              "key": "d",
+              "name": "Format document with...",
               "type": "command"
             },
             {
-              "command": "excel.preview",
-              "key": "V",
-              "name": "Excel preview",
+              "command": "editor.action.formatChanges",
+              "key": "m",
+              "name": "Format modified lines only",
+              "type": "command"
+            },
+            {
+              "command": "editor.action.formatSelection",
+              "key": "s",
+              "name": "Format selection",
+              "type": "command"
+            },
+            {
+              "command": "editor.action.formatSelection.multiple",
+              "key": "S",
+              "name": "Format selection with...",
               "type": "command"
             }
           ],
-          "key": "c",
-          "name": "CSV/Excel...",
+          "key": "=",
+          "name": "Format...",
           "type": "bindings"
         },
         {
           "bindings": [
             {
-              "command": "markdown.extension.editing.toggleMath",
-              "key": "m",
-              "name": "Toggle math",
+              "command": "go.fill.struct",
+              "key": "f",
+              "name": "Fill struct",
               "type": "command"
             },
             {
-              "command": "markdown.extension.checkTaskList",
-              "key": "c",
-              "name": "Check Task List",
+              "command": "go.add.tags",
+              "key": "t",
+              "name": "Add tags to struct fields",
               "type": "command"
             },
             {
-              "command": "markdown.extension.editing.toggleBold",
-              "key": "b",
-              "name": "Toggle bold",
+              "command": "go.remove.tags",
+              "key": "T",
+              "name": "Remove tags from struct fields",
               "type": "command"
             },
             {
-              "command": "markdown.extension.editing.toggleItalic",
+              "command": "go.import.add",
               "key": "i",
-              "name": "Toggle italic",
+              "name": "Add import",
               "type": "command"
             },
             {
-              "command": "markdown.extension.editing.toggleStrikethrough",
-              "key": "s",
-              "name": "Toggle strikethrough",
+              "command": "go.impl.cursor",
+              "key": "I",
+              "name": "Generate interface stubs",
+              "type": "command"
+            }
+          ],
+          "key": "i",
+          "name": "Insert/remove...",
+          "type": "bindings"
+        },
+        {
+          "bindings": [
+            {
+              "command": "go.languageserver.restart",
```

Comment:
I thought about this too, it doesn't seem like there is a consistent naming across different layers. For example, Python layer uses `v` to represent environment (I assume from `venv`), C# layer has `s` to represent OmniSharp (its language server). If `b` more common, maybe we should just use `b`?

Reply:
`b` is different from environment (I think). I was talking just about this key binding.
I was ok with the other `SPC e` section.
But I am not a go developer as I said.
`b` is related to the language server protocol, I think we should use the lsp key bindings everywhere because they are common on almost every language (I think).

Reply:
Ah got you, that makes sense

Reply:
Fair - I was using `e` since it corresponded with "environment", and I made the stretch of `SPC f e R` (restarting your editor config) to be synonymous with "restarting something" (in this case, restarting the language server). If this is a typical keybinding for LSP, I'll move over to using that, but does that mean everything else should move over there as well? Would feel like a weird split I think.

Reply:
Unless I hear a strong objection, I'm just going to change the `e` menu to `b`, since that's more in-line with what the LSP docs expect. Will push that shortly.